jazz_deployments/components/validation/validate_create_payload.js

```diff
@@ -0,0 +1,139 @@
+// =========================================================================
```

Comment:
Why do we have multiple validation files one for create, update, list and rebuild. 
Cant we have just one validation file which is generic enough to handle all 4 scenarios. something that can be parameterized.

Reply:
why not?

Reply:
The acceptance criteria for this story will be met w/o this refactoring, so am ok with accepting this story w/o this specific change.